style.css

```diff
@@ -603,16 +618,48 @@ body {
     background-color: #f4f1f0;
     cursor: not-allowed;
   }
+
+  #desk_ver {
+    display: none;
+  }
+
+  #first-name {
+    display: none;
+  }
+
+  #last-name {
+    display: none;
+  }
+
+  #sm1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-top: 5%;
+    width: 200px;
+    height: 40px;
+    margin-bottom: 30%;
+    align-items: center;
+    padding-right: 0%;
+    margin-right: 0%;
+  }
+
+  .text4 {
+    justify-content: center;
+    margin: 0% 50px;
+  }
 }
 
 @media (min-width: 765px) {
+  body
   /* Background color */
-  body {
+  #back {
     background-image: url("images/Illustration-header.png");
     background-repeat: no-repeat;
     background-origin: content-box;
-    background-position: 100% -10%;
-    background-size: 35%;
+    background-position: 100% -0.5%;
+    background-attachment: scroll;
+    background-size: 500px;
     display: flex;
     flex-direction: column;
   }
```

Comment:
Greeting!

You're doing a great work, but according to the Figma design, there are a few things you'll need to change to go to the next steps.

- Kindly, I suggest you stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

> ## The Figma design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 31 24 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/7ae9262d-561d-45db-976f-966a66e6a74e">
>
> ## Your Design
>
><img width="1440" alt="Screenshot 2023-06-15 at 11 04 20 PM" src="https://github.com/Rdzaman20000/Portfolio/assets/91237525/3b082a9f-5999-4d6c-8293-30dac7c54735">